JSDemos/Demos/DataGrid/ExcelJSExportImages/Vue/data.ts

```diff
@@ -1,4 +1,20 @@
-export const employees = [{
+export type Employee = {
```

Comment:
Can use type or interface type everywhere? Instead class

Reply:
I use _type_ everywhere, because classes require to set default values to each property.